compte/views.py

```diff
@@ -149,6 +154,30 @@ def change_email(request, activation_token):
         )
 
 
+@login_required
+def delete_account(request):
+    try:
+        res = remove_personal_informations(request.user, shortuuid.uuid)
+    except RuntimeError as err:
+        messages.add_message(
+            request,
+            messages.WARNING,
+            f"Erreur lors de la désactivation du compte: {err}",
```

Comment:
How about not rendering the technical exception message to the end user? And how about logging the error instead, so Sentry can be notified?